todo_app/Data/mongo_items.py

```diff
@@ -45,6 +48,8 @@ def add_item(self, title):
 
         self.get_db().collection.insert_one({'status': 'To Do', 'name': title})
 
+        log.info('add_item: title[%s]', title)
```

Comment:
This feels like a really natural log to have, and possibly it is, but it's worth considering how easily this could start adding personally-identifiable info into the logs!